libs/dh/market-participant/actors/feature-delegation/src/lib/util/dh-group-delegations.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { DelegationMessageType } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import { DhDelegations } from '../dh-delegations';
+import { dhGroupDelegations } from './dh-group-delegations';
+
+describe(dhGroupDelegations.name, () => {
+  it('should group delegations by direction and type', () => {
```

Comment:
Praise: Nice you are adding tests

Reply:
Copilot helpe with auto generating them. This makes util functions are quite easy to test.